apps/election-manager/src/screens/TallyReportScreen.tsx

```diff
@@ -126,6 +130,16 @@ const TallyReportScreen: React.FC = () => {
       fileSuffix = `scanner-${scannerId}`
       return `${statusPrefix} Scanner Tally Report for Scanner ${scannerId}`
     }
+    if (batchId) {
+      const batchTally = filterTalliesByParams(fullElectionTally!, election, {
+        batchId,
+      }) as BatchTally
```

Comment:
I think you're safe in this case because you're asking for a `batchId` that you know is present in at least one CVR entry, but in general `filterTalliesByParams` is not guaranteed to return a `BatchTally` even when only filtering by `batchId` since it could return a `Tally` from `getEmptyTally`. You could address this by overloading `filterTalliesByParams` to include a narrower signature with only `{ batchId: string }` that returns `BatchTally`, and a `getEmpyBatchTally` function for that case. Or we can not worry about it for now since I think this is safe for the moment.